src/plugins/link.js

```diff
@@ -0,0 +1,55 @@
+import { generateUrl } from '@nextcloud/router'
+import { Plugin } from 'prosemirror-state'
+import markdownit from './../markdownit'
+
+const clickHandler = ({ editor }) => {
+	return new Plugin({
+		props: {
+			handleClick: (view, pos, event) => {
+				const linkElement = event.target.parentElement instanceof HTMLAnchorElement
+					? event.target.parentElement
+					: event.target
+
+				const isLink = linkElement && linkElement instanceof HTMLAnchorElement
+
+				const htmlHref = linkElement?.href
```

Comment:
I've changed this part of the original code to make it possible to run tests inside Cypress.

Originally we used `editor.getAttributes('link')` but it not works in cypress

When we interact natively/manually with the editor, the original events "bubble" from the `span` tag, but in cypress tests, the event comes from `a` tag, and this tag doesn't have attributes in the editor.
It makes the automated test impossible to work, because of this I've changed the code to use the native properties of the link.